keyboards/rura66/rev1/config.h

```diff
@@ -0,0 +1,163 @@
+ *
+ * Change this to how you wired your keyboard
+ * COLS: AVR pins used for columns, left to right
+ * ROWS: AVR pins used for rows, top to bottom
+ * DIODE_DIRECTION: COL2ROW = COL = Anode (+), ROW = Cathode (-, marked on diode)
+ *                  ROW2COL = ROW = Anode (+), COL = Cathode (-, marked on diode)
+ *
+ */
+#define MATRIX_ROW_PINS { D4, C6, D7, E6, B4 }
+#define MATRIX_COL_PINS { F4, F5, F6, F7, B1, B3, B2 }
+#define UNUSED_PINS
+
+/* COL2ROW, ROW2COL*/
+#define DIODE_DIRECTION COL2ROW
+
+/* Split hand configration */
+#define SPLIT_HAND_MATRIX_GRID D7,B2
+#define SPLIT_HAND_MATRIX_GRID_LOW_IS_RIGHT
+
+/* Dip switch on matrix grid */
+#define DIP_SWITCH_MATRIX_GRID { {0,6}, {1,6}, {5,6}, {6,6} }
+
+/*
+ * Split Keyboard specific options, make sure you have 'SPLIT_KEYBOARD = yes' in your rules.mk, and define SOFT_SERIAL_PIN.
+ */
+#define SOFT_SERIAL_PIN D2  // or D1, D2, D3, E6
+
+#define BACKLIGHT_PIN D3
+#define RGB_DI_PIN D3
+#define RGBLED_NUM 66 // Number of LEDs
+
+#ifdef RGB_MATRIX_ENABLE
+  #define DRIVER_LED_TOTAL RGBLED_NUM
+  #define RGB_MATRIX_KEYPRESSES // reacts to keypresses
+//  #define RGB_MATRIX_KEYRELEASES // reacts to keyreleases (instead of keypresses)
+//  #define RGB_DISABLE_AFTER_TIMEOUT 0 // number of ticks to wait until disabling effects
+  #define RGB_DISABLE_WHEN_USB_SUSPENDED true // turn off effects when suspended
+  #define RGB_MATRIX_FRAMEBUFFER_EFFECTS
+//  #define RGB_MATRIX_LED_PROCESS_LIMIT (DRIVER_LED_TOTAL + 4) / 5 // limits the number of LEDs to process in an animation per task run (increases keyboard responsiveness)
+//  #define RGB_MATRIX_LED_FLUSH_LIMIT 16 // limits in milliseconds how frequently an animation will update the LEDs. 16 (16ms) is equivalent to limiting to 60fps (increases keyboard responsiveness)
+  #define RGB_MATRIX_MAXIMUM_BRIGHTNESS 150 // limits maximum brightness of LEDs to 150 out of 255. Higher may cause the controller to crash.
+  #define RGB_MATRIX_HUE_STEP 8
+  #define RGB_MATRIX_SAT_STEP 8
+  #define RGB_MATRIX_VAL_STEP 8
+  #define RGB_MATRIX_SPD_STEP 10
+#endif
+
+#ifdef RGBLIGHT_ENABLE
+  #define RGBLIGHT_SPLIT
+  #define RGBLED_SPLIT { 33, 33 }
+  #define RGBLIGHT_LIMIT_VAL 120 /* The maximum brightness level */
+//  #define RGBLIGHT_SLEEP  /* If defined, the RGB lighting will be switched off when the host goes to sleep */
+   /*== all animations enable ==*/
+//  #define RGBLIGHT_ANIMATIONS
+  /*== or choose animations ==*/
+//  #define RGBLIGHT_EFFECT_BREATHING
+  #define RGBLIGHT_EFFECT_RAINBOW_MOOD
+  #define RGBLIGHT_EFFECT_RAINBOW_SWIRL
+//   #define RGBLIGHT_EFFECT_SNAKE
+//   #define RGBLIGHT_EFFECT_KNIGHT
+//   #define RGBLIGHT_EFFECT_CHRISTMAS
+  #define RGBLIGHT_EFFECT_STATIC_GRADIENT
+  #define RGBLIGHT_EFFECT_RGB_TEST
+//   #define RGBLIGHT_EFFECT_ALTERNATING
+#endif
+
+/* Custom font */
+#define OLED_FONT_H "keyboards/rura66/common/glcdfont.c"
+
+/* Encorder */
+#define ENCODERS_PAD_A { B6 }
+#define ENCODERS_PAD_B { B5 }
```

Comment:
`rura66/rev1/rules.mk` declares `ENCODER_ENABLE = no`.  If so, this declaration is unnecessary.

`rura66/rev1/rules.mk` では、`ENCODER_ENABLE = no` と宣言されています。  そうであるならば、この宣言は不要です。

```suggestion
// #define ENCODERS_PAD_A { B6 }
// #define ENCODERS_PAD_B { B5 }
```